lib/contracts/compiler.js

```diff
@@ -16,15 +16,16 @@ class Compiler {
 
     let pluginCompilers = self.plugins.getPluginsProperty('compilers', 'compilers');
     pluginCompilers.forEach(function (compilerObject) {
-      if(available_compilers[compilerObject.extension] === undefined){
-        available_compilers[compilerObject.extension] = compilerObject.cb;
+      if(available_compilers[compilerObject.extension] == undefined){
```

Comment:
If you mean `if(not truthy)` (not null, empty, undefined, false), you can just do `if(!available_compilers[compilerObject.extension])`